.github/workflows/cbmc-update.yml

```diff
@@ -0,0 +1,48 @@
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+name: Attempt CBMC update
+
+on:
+  schedule:
+    - cron: "30 4 * * Mon" # Run this every Monday at 04:30 UTC
+  workflow_dispatch:     # Allow manual dispatching for a custom branch / tag.
+
+permissions:
+  checks: write
+  contents: write
+  pull-requests: write
+
+jobs:
+  create-cargo-update-pr:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout Kani
+        uses: actions/checkout@v3
+
+      - name: Compare CBMC versions
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          grep ^CBMC_VERSION kani-dependencies >> $GITHUB_ENV
+          CBMC_LATEST=$(gh -R diffblue/cbmc release list | grep Latest | awk '{print $1}' | cut -f2 -d-)
+          echo "CBMC_LATEST=$CBMC_LATEST" >> $GITHUB_ENV
+          if [ x$CBMC_LATEST != x$CBMC_VERSION ] ; then
```

Comment:
Can you please log the two versions to the terminal please? I think it's very hard to debug these jobs since they don't leave enough logs.

Reply:
There now is the `git diff` plus the version numbers are in the commit message or issue report (whichever might apply).